src/test/java/com/google/jenkins/plugins/computeengine/ComputeEngineCloudIT.java

```diff
@@ -212,14 +213,16 @@ public void testWorkerCreated() throws Exception {
         assertEquals(logs(),3, i.getLabels().size());
 
         // Instance should have a label with key CONFIG_LABEL_KEY and value equal to the config's name prefix
-        assertEquals(logs(), validInstanceConfiguration1().namePrefix, i.getLabels().get(ComputeEngineCloud.CONFIG_LABEL_KEY));
+        assertEquals(logs(), ic.namePrefix, i.getLabels().get(ComputeEngineCloud.CONFIG_LABEL_KEY));
         assertEquals(logs(), String.valueOf(cloud.name.hashCode()), i.getLabels().get(ComputeEngineCloud.CLOUD_ID_LABEL_KEY));
     }
     
     @Test(timeout = 300000)
     public void test1WorkerCreatedFor2Executors() throws Exception {
+        logOutput.reset();
+
         ComputeEngineCloud cloud = (ComputeEngineCloud) r.jenkins.clouds.get(0);
-        cloud.addConfiguration(validInstanceConfiguration1());
+        cloud.addConfiguration(validInstanceConfiguration1("2"));
```

Comment:
Using hard coded literal like this should be avoided as it doesn't give code readers much context. Consider either using a constant with a descriptive name, or else an in-line comment describing the value being passed.

Reply:
Revised.